docs/modules/usage/how-to/custom-sandbox-guide.md

```diff
@@ -12,53 +12,131 @@ You have two options for customization:
 1. Use an existing image with the required software.
 2. Create your own custom Docker image.
 
-If you choose the first option, you can skip the `Create Your Docker Image` section.
+## Important Requirements
 
-## Create Your Docker Image
+Before proceeding with either option, note these key requirements:
+
+1. The base image must be Debian-based
+2. The user in the container MUST be `root` for proper functionality
+3. Any installed packages should be available system-wide, not just for specific users
 
-To create a custom Docker image, it must be Debian based.
+## Create Your Docker Image
 
-For example, if you want OpenHands to have `ruby` installed, create a `Dockerfile` with the following content:
+Here's a complete example of a custom runtime Dockerfile:
 
 ```dockerfile
 FROM debian:latest
```

Comment:
```suggestion
FROM python-nodejs:python3.12-nodejs22
```
Rationale: Using the same base as the default sandbox image and just adding ruby makes it a bit more easy to get the right versions of python and node are being installed in the runtime image, rather than relying on the python and node versions that are included in debian:latest